package.json

```diff
@@ -2,12 +2,13 @@
   "name": "gmpnodejs2021q3",
   "version": "1.0.0",
   "description": "study",
-  "type": "module",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "task1": "nodemon task1.js",
-    "task2": "nodemon task2.js"
+    "lint": "eslint . --ext .ts",
+    "task1": "nodemon --exec \"npm run tsnode1\"",
```

Comment:
*nitpick*: nodemon has a ability to be configured by config-file or package.json section
good example can be found [here](https://stackoverflow.com/questions/37979489/how-to-watch-and-reload-ts-node-when-typescript-files-change)